_posts/2021-11-30-best-practises-sprint-demo.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Sprint Demo:Best Practises"
+excerpt: "To ensure we follow best practises in ensuring successful sprint demo for the customer."
+tags: Sprint Demo
+authors:
+- Venkata Gopu
+header:
+  teaser: /assets/images/post/sprintdemo.jpeg
+  teaser_alt: Fragments
+category: Software Quality Assurance
+---
+
+
+Here are a few tips:.
+
+{% include
+  components/figure.html
+  url="/assets/images/post/SprintReview.jpeg"
+  description="Sprint Review"
+%}
+
+### A Sprint Review is an essential part of the Sprint process. It’s an opportunity for the development team to show off its work and to excite stakeholders.
+
+
+#### Agenda template
```

Comment:
This is rather a presentation format, I'd expect something different for a blog post

Reply:
Especially considering this is public, I think so too.